lib/recurly/bank-account.js

```diff
@@ -92,6 +93,10 @@ function token (options, done) {
     inputs.type = 'iban_bank_account';
   }
 
+  if ('branch_code' in inputs) {
+    inputs.type = 'bacs_bank_account';
+  }
```

Comment:
For this, to avoid making the coercion too complex as we add additional bank account types, I think we should start having merchants specify the `scheme_type` in their `token` call.

So in this method, we can accept `scheme_type` and cast it to `inputs.type` or accept `type` and allow it to pass into `inputs` directly.

That way we can move away from the black box decision-making we're already kinda suffering from in casting token type intent.

Reply:
to round this up we are going to do "type" directly and coerce it to what RJS wants